connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/utils/ProtobufOptions.scala

```diff
@@ -46,6 +46,41 @@ private[sql] class ProtobufOptions(
   // record has more depth than the allowed value for recursive fields, it will be truncated
   // and corresponding fields are ignored (dropped).
   val recursiveFieldMaxDepth: Int = parameters.getOrElse("recursive.fields.max.depth", "-1").toInt
+
+  // Whether to render fields with zero values when deserializing protobufs to a spark struct.
+  // When a field is empty in the serialized protobuf, this library will deserialize them as
+  // null by default. However, this flag can control whether to render the type-specific zero value.
+  // This operates similarly to `includingDefaultValues` in java's jsonformat, or `emitDefaults` in
```

Comment:
nit: protobuf-java-util's rather than java's? it is not Java library. 
Also, `JsonFormat` (with capital letter). 

Alternately we could say `com.google.protobuf.util.JsonFormat` and avoid saying `java's jsonformat`.

Reply:
done!